modules/virt-enabling-dedicated-resources.adoc

```diff
@@ -22,12 +22,12 @@ endif::[]
 = Enabling dedicated resources for a {object}
 
 You can enable dedicated resources for a {object} in the
-*{object-gui} Overview* page of the web console.
+*Details* tab. Virtual machine templates that were  created by using a Red Hat template or the interactive wizard can be enabled with dedicated resources.
 
 .Procedure
 
 . Click *Workloads* -> *{object-gui}s* from the side menu.
-. Select a {object} to open the *{object-gui} Overview* page.
+. Select a {object} to open the *{object-gui}* tab.
 . Click the *Details* tab.
```

Comment:
Is this step necessary? Selecting a template takes us directly to the Details screen, just like for VMs it takes us to the Overview screen.

Reply:
Dedicated resources apply VMs only. Therefore, these instructions to select a VM, go to the Overview page, and click on the Details tab would hold up as being true. I corrected the introductory text to ensure that it is referring to VMs and not VM templates (it was referring to VM templates, and that was not correct):

You can enable dedicated resources for a {object} in the
*Details* tab. Virtual machines that were created by using a Red Hat template or the interactive wizard can be enabled with dedicated resources.

Thanks for pointing out!